markets/spot-market/contracts/storage/AsyncOrder.sol

```diff
@@ -2,12 +2,15 @@
 pragma solidity >=0.8.11 <0.9.0;
 
 import "@synthetixio/core-modules/contracts/interfaces/ITokenModule.sol";
+import "@synthetixio/core-contracts/contracts/utils/DecimalMath.sol";
 import "../utils/SynthUtil.sol";
 
 /**
  * @title Async order top level data storage
  */
 library AsyncOrder {
+    error InvalidSharesCalculation(uint256 expected, uint256 actual);
```

Comment:
this error name could be more descriptive to an innocent user. for example something like "InsufficientAmountDeposited" or something like that, could perhaps be more helpful

Reply:
makes sense. changing